doc/api/cli.md

```diff
@@ -2018,6 +2018,35 @@ On a machine with 2 GiB of memory, consider setting this to
 $ node --max-old-space-size=1536 index.js
 ```
 
+### `--max-semi-space-size=SIZE` (in megabytes)
+
+Sets the maximum [semi-space][] size for V8's
+[scavenge garbage collector][] in MiB.
+Increasing the max size of a semi-space may
+improve throughput for Node.js at the cost of more
+memory consumption.
+Since the young generation size of the V8 heap is three times
+(see [`YoungGenerationSizeFromSemiSpaceSize`][] in V8)
+the size of the semi-space, an increase of 1 MiB to semi-space applies
+to each of the three individual semi spaces and
+causes the heap size to increase by 3 MiB.
+The throughput improvement depends on your workload
+(see [#42511](https://github.com/nodejs/node/issues/42511)).
+
+The default value is
+16 MiB for 64-bit systems and 8 MiB for 32-bit systems.
+To get the best configuration for your application,
+you should try different max-semi-space-size values
+when running benchmarks for your application.
```

Comment:
Nit: The very short line lengths make this a bit more difficult to read.

```suggestion
Sets the maximum [semi-space][] size for V8's [scavenge garbage collector][] in MiB.
Increasing the max size of a semi-space may improve throughput for Node.js at the
cost of more memory consumption.

Since the young generation size of the V8 heap is three times (see
[`YoungGenerationSizeFromSemiSpaceSize`][] in V8) the size of the semi-space, an
increase of 1 MiB to semi-space applies to each of the three individual semi spaces
and causes the heap size to increase by 3 MiB. The throughput improvement depends
on your workload (see [#42511](https://github.com/nodejs/node/issues/42511)).

The default value is 16 MiB for 64-bit systems and 8 MiB for 32-bit systems. To get the
best configuration for your application, you should try different max-semi-space-size
values when running benchmarks for your application.
```

Reply:
Thanks @jasnell, it looks better after adjust the line lengths.